cg/services/deliver_files/deliver_files_service/deliver_files_service_factory.py

```diff
@@ -126,14 +126,14 @@ def _validate_delivery_type(delivery_type: DataDelivery):
         )
 
     def build_delivery_service(
-        self, workflow: Workflow, delivery_type: DataDelivery
+        self, workflow: Workflow, delivery_type: DataDelivery, app_tag: str
```

Comment:
we don't need workflow anymore
```suggestion
        self, delivery_type: DataDelivery, app_tag: str
```